inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_detailed.tpl

```diff
@@ -100,4 +113,43 @@
 </div>
 </div>
 
-{% endblock body %}
\ No newline at end of file
+{% endblock body %}
+
+{% block javascript %}
+  {{ super() }}
+  <script type="text/javascript">
+    // require(
+    //   [
+    //     "jquery",
+    //     "js/plots"
+    //   ],
+      
+    //   function(
+    //     $,
+    //     Plots
+    //   ) {
+        
+    //     $(function() { 
+    //       Plots.attachTo(document, context);
+    //     });
+
+    // });
+
```

Comment:
Yes, for a `require` call to work in the Jinja template it first needs to be part of a bundle (like adding it to `inspire_base_init.js`) You can check how we were doing before and we might need to create a JS bundle for the detail view?